### DISCUSSION TITLE
Set color in my app is not the same in the modelviewer editor.

### DISCUSSION THREAD

**Discussion by AYTSistemas13** [google/model-viewer#3908] [Q&A] [answered]
When I set the color with the function modelViewerColor.model?.materials[29].pbrMetallicRoughness.setBaseColorFactor(colorString);
Is not the same that I saw and pick in the editor in modelviewer.dev.
Here is a screenshot in the modelviewer.dev editor:
![capt1](https://user-images.githubusercontent.com/80072081/198333694-b08d0fd9-83ba-46da-9dc7-2048db0f043e.PNG)
And here is a screenshot in my project, the color is the same in RGBA format but for some reason when the app display the color is more soft:
![capt2](https://user-images.githubusercontent.com/80072081/198334434-45c0f9c2-4c5d-42e8-b4af-4b24e8c63038.PNG)
I tried everything and I can´t fix this, I don´t know if i´m doing something wrong.

**Comment by elalish** [comment, marked as answer]
Same issue as #3906.

**Reply by AYTSistemas13** [reply]
Ok I got it but in the next function when I set the color in my case is:
modelViewerColor.model?.materials[29].pbrMetallicRoughness.setBaseColorFactor(colorString);
.setBaseColorFactor(colorString) expects an element RGBA, how or in which part of my code I need to use this function by three js .convertSRGBToLinear ().
My model-viewer is part of an Angular project.
What I need to do in this case?

**Reply by AYTSistemas13** [reply]
I forgot to mention in the principal issue that I already divided the first three numbesr of rgba code color by 255, in the example I passed in a function the complete decimal number for example (.38712737123812,.182093809182093,.182318093123,1)  and still the color was not the same.

**Reply by AYTSistemas13** [reply]
For now I just use this online tool, https://ajalt.github.io/colormath/converter/ to put the rgb color and then the tool displays different formats and I took the color codes from Linear sRGB and now the color match with the model viewer editor. Thanks.
